readthedocs.yml

```diff
@@ -0,0 +1,31 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
```

Comment:
This should be renamed: `.readthedocs.yaml`

Reply:
✓ So renamed